src/ralph/back_office/models.py

```diff
@@ -84,6 +84,7 @@ class BackOfficeAssetStatus(Choices):
     sale = _("sale")
     loan_in_progress = _("loan in progress")
     return_in_progress = _("return in progress")
+    in_quarantine = _("in quaraintine")
```

Comment:
I'm not sure about a new status. What is a purpose of ``in quarantine``? Maybe you should create another type for SIM Card?

Reply:
When the card is returned,  it must go into quarantine mode. For ex. exec leave the company. This card sim must have quarantine, because other supplier can ring to new owner. New owner (for ex. developer) get unwanted phones. On the other hand, the time of quarantine is different  (depending on the position). I would like to filter SIM Card which is "in quarantine"

Reply:
sure - but is it the sim card which is in quarantine mode or back office asset?

Reply:
After discussion we dedicated "in quarantine" mode its only for SIMCard and I added 
unique choices for SIMCard models.